spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/catalog/GravitinoCatalog.java

```diff
@@ -336,4 +351,27 @@ private String getDatabase(NameIdentifier gravitinoIdentifier) {
         "Only support 3 level namespace," + gravitinoIdentifier.namespace());
     return gravitinoIdentifier.namespace().level(2);
   }
+
+  @VisibleForTesting
+  static com.datastrato.gravitino.rel.TableChange transformTableChange(TableChange change) {
+    if (change instanceof TableChange.SetProperty) {
+      return transformSetProperty((TableChange.SetProperty) change);
+    } else if (change instanceof TableChange.RemoveProperty) {
+      return transformRemoveProperty((TableChange.RemoveProperty) change);
+    } else {
+      throw new UnsupportedOperationException(
+          String.format("Unsupported table change %s", change.getClass().getName()));
+    }
+  }
+
+  private static com.datastrato.gravitino.rel.TableChange transformSetProperty(
```

Comment:
I prefer move the code from `transformSetProperty` and `transformRemoveProperty` to `transformTableChange` because they are too simple.

Reply:
Fixed